plasmapy/diagnostics/charged_particle_radiography.py

```diff
@@ -1088,11 +1107,140 @@ def run(
             f"{self.fract_deflected*100}%"
         )
 
+        # Simulation has not run, because creating new particles changes the simulation
+        self._has_run = True
+
+    @property
+    def results_dict(self):
+        r"""
+        A dictionary containing the results of the simulation.
+
+        .. list-table:: Dictionary keys and descriptions.
+           :width: 100%
+           :widths: 1 1 4
+           :header-rows: 1
+
+           * - Key
+             - Type
+             - Description
+           * - ``"source"``
+             - `~numpy.ndarray`
+             - The source location vector, in meters.
+           * - ``"detector"``
+             - `~numpy.ndarray`
+             - The detector location vector, in meters.
+           * - ``"mag"``
+             - `float`
+             - The system magnification.
+           * - ``"nparticles"``
+             - `int`
+             - Number of particles in the simulation.
+           * - ``"max_deflection"``
+             - `~astropy.units.Quantity`
+             - The maximum deflection experienced by a particle in the
+               simulation, in radians.
```

Comment:
Since the rest of the values in this dictionary are numpy arrays, I'm thinking we drop units for ``max_deflection``.  We can do this either at the `max_deflection` property or just when building the `results_dict`.  I feel this will make the whole `results_dict` more consistent and, while reviewing `test_saving_ouput`, the `numpy` saving and loading system basically converts the `Quantity` back to a numpy array.

Reply:
This section in the docstring needs to be updated to reflect that `max_deflection` is no longer a `Quantity` in the dictionary.